src/toggle-switch/toggle-switch.scss

```diff
@@ -2,194 +2,212 @@
 // See LICENSE.md in the project root for license terms and full copyright notice.
 @import '../style/index';
 
+$toggle-switch-handle: $iui-icons-default;
+
 @mixin iui-toggle-switch {
   @include iui-reset;
   display: flex;
   align-items: center;
+  gap: $iui-s;
+  align-items: center;
   font-size: $iui-font-size;
   width: fit-content;
-  padding: ($iui-baseline/2) 0;
```

Comment:
> Removed padding surrounding toggle switch (https://github.com/iTwin/iTwinUI/issues/315). Spacing is handled by .iui-input-group. If toggle is by itself, spacing must be added by the user.

I am not sure it is ok to do this as I bet more users are using toggle switches alone and that's why I decided to leave it for major release:
![image](https://user-images.githubusercontent.com/36186912/154910891-1ab98fa2-0c3c-43b8-ad65-9689022064a1.png)

Reply:
The only way I can think to resolve this is to add a modifier class similar to our typography components' `.iui-text-spacing`.

Reply:
Worth pointing out that this is how our checkboxes / radios work without `.iui-labeled-input` wrapping them.  Our demos don't have them touching because we have a 11px `gap` on the `<section>`s.

Reply:
I started to think maybe it is ok. This is kind of fixing a bug 😀